templates/lab-results/templates/header.tpl

```diff
@@ -0,0 +1,45 @@
+{{define "info-field"}}
+    <table-cell>
+        <paragraph margin="0">
+            <text-chunk color="#1B7A98" font-size="12">{{ .Title }}</text-chunk>
+        </paragraph>
+    </table-cell>
+    <table-cell>
+        <paragraph>
+            <text-chunk font-size="12">{{ .Value }}</text-chunk>
+        </paragraph>
+    </table-cell>
+{{end}}
+
+<division margin="10">
+    <table columns="2" column-width="0.5 0.5">
```

Comment:
No need to specify the `column-widths` attribute if all the columns need to be equal.